example/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "example",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run compile && node index.js",
+    "compile": "faucet --no-fingerprint --compact"
+  },
+  "author": "Falk Hoppe <falkhoppe81@gmail.com>",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "complate-express": "file:..",
+    "express": "^4.15.3"
+  },
+  "devDependencies": {
+    "complate-jsx": "^0.7.1"
```

Comment:
this definetly is a thing to be discussed, because often the same CI environment which runs the bundle process, is already running in production mode. But this is not important for that example application. And yes this dependency is not necessary once the bundle is created, so it should not be delivered to production surroundings.